ee/clickhouse/queries/test/test_cohort_query.py

```diff
@@ -325,6 +325,83 @@ def test_performed_event_lte_1_times(self):
 
         self.assertEqual(set([p2.uuid]), set([r[0] for r in res]))
 
+    def test_performed_multiple_performed_event_multiple(self):
```

Comment:
nit: Kinda confusing name. Maybe it should be `test_can_handle_many_performed_multiple_filters`?